plugins/module_utils/hmc_resource.py

```diff
@@ -68,6 +68,29 @@ def pingTest(self, i_host):
 
         return result
 
+    def sshTest(self, i_host, u_host):
+        pattern = re.compile(r"Alive")
+        report = ("No response", "Alive")
+        if None is u_host:
+            cmd = "ssh -o ConnectTimeout=5 -o Batchmode=yes " + i_host.strip() + " echo 'Alive'"
```

Comment:
How it handle ssh connection without user? Does it not use the control user as HMC user?

Reply:
Yes, it uses control user, I made this way because customer who suggested the code is not referring the username so, just to support that case as well I added that condition.